localstack/testing/pytest/fixtures.py

```diff
@@ -922,6 +922,7 @@ def deploy_cfn_template(
     is_change_set_finished,
 ):
     state = []
+    specified_max_wait = None
```

Comment:
not sure I understand the reason why we would keep this at this level and potentially update it with each deploy call? :thinking:

Reply:
Ah, good catch, my bad - was reading the variable scoping incorrectly before. `max_wait` is already available to the `_destroy_stack` function. 👍 Reverted/adjusted the change, thanks for the pointer!